moveit_core/robot_model/src/robot_model.cpp

```diff
@@ -47,17 +47,19 @@
 #include "order_robot_model_items.inc"
 
 /* ------------------------ RobotModel ------------------------ */
```

Comment:
this comment above is odd, can you remove that too please?

Reply:
deleted the comment.